### ISSUE TITLE
Missing ViewNavigationArgs on back navigation

### ISSUE THREAD

**Issue by weases** [exrin/Exrin#1]
**Observed Behaviour**
On performing a back navigation via navigationService.GoBack(parameter) method, the parameter is not available in OnBackNavigated(object args) callback. Unlike for forward navigation it is always null.

**Possible fix**
Probably it is sufficient to add 

```object Parameter { get; set; }```

to IViewNavigationArgs interface and modify the call to model.OnBackNavigated in BaseStack implementation

```
private void proxy_OnPopped(object sender, IViewNavigationArgs e)
{
   ...
   if (e.CurrentView != null)
   {
	var model = e.CurrentView.BindingContext as IViewModel;
	if (model != null)
	     model.OnBackNavigated(e.Parameter);
   }
   ...
}
```

**Comment by adamped** [comment]
@weases - thanks for letting me, excellent catch. I have just fixed this up and released 1.0.4-beta1, along with a few other small feature enhancements. Let me know if this fixes your issue. 

https://www.nuget.org/packages/Exrin/1.0.4-beta1

**Comment by weases** [comment]
@adamped This works as expected now. Thank you for the fast reply.
